editor/components/table-of-contents/style.scss

```diff
@@ -1,5 +1,10 @@
 .table-of-contents__popover.components-popover:not(.is-mobile) .components-popover__content {
 	min-width: 380px;
+	height: auto;
+	overflow-y: auto;
+	@include break-medium {
```

Comment:
Would this mean that it is not scrollable for smaller displays?

Reply:
Maybe we should make this popover fullscreen on mobile (like the inserter) using the `expandOnMobile` prop.

Reply:
To see Table of Contents on smaller display you need to access Settings > Document Tab > Table of Contents.

![screenshot from 2018-01-10 08-30-33](https://user-images.githubusercontent.com/6010232/34767933-8fb7a4a4-f5e0-11e7-9e39-b3fb3202dec2.png)

The button of Table of Contents is set to not display (../gutenberg/editor/edit-post/header/header-toolbar/style.scss)

![screenshot from 2018-01-10 08-31-36](https://user-images.githubusercontent.com/6010232/34767997-c3449a02-f5e0-11e7-96ef-4cf446dac50d.png)

Reply:
Thanks so much for the contribution!

Given #4287 and the work and discussion that's ongoing in #4109, we might be looking at removing the Table of Contents panel from the sidebar entirely, in favor of just the popover. 

Given that, it seems like expanding to be modal on mobile is a great solution.